src/pages/Missions/Missions.tsx

```diff
@@ -87,6 +88,14 @@ export const useGetMissions = () => {
         coverImage: YatBg,
         onClick: () => window.open('https://fantasy.y.at/invite/yduad7mm'),
       },
+      {
+        title: translate('missions.foxRewards.title'),
+        subtitle: translate('missions.foxRewards.subtitle'),
+        buttonText: translate('missions.foxRewards.cta'),
+        coverImage: FoxRewardsBg,
+        startDate: '2025-01-01 7:00 AM',
+        onClick: () => window.open('https://app.shapeshift.com'),
```

Comment:
Since this is a hook we could probably route to `/` using react-router?

Reply:
Tested and discussed this one out, not clickable as a "Coming Soon" card and there's never going to be an intent for this